dsaT3/filplot_funcs.py

```diff
@@ -607,27 +596,55 @@ def plot_fil(fn, dm, ibox, multibeam=None, figname_out=None,
         g.create_dataset('probability',data=[prob])        
         g.close()
     
-        
     not_real = plotfour(dataft, dataft.mean(0), datadm, datadm0=datadm0, 
-                        beam_time_arr=beam_time_arr, figname_out=figname_out, dm=dm,
+                        beam_time_arr=beam_time_arr, figname=figname, dm=dm,
                         dms=[dms[0],dms[-1]], 
                         suptitle=suptitle, heimsnr=heimsnr,
                         ibox=ibox, ibeam=ibeam, prob=prob,
                         showplot=showplot,
                         multibeam_dm0ts=multibeam_dm0ts,fnT2clust=fnT2clust,imjd=imjd,
                         fake=fake)
 
-    return not_real,prob
-    
+    return not_real, prob
+
 
-def filplot_entry(datestr,trigger_dict,
-                  toslack=True,classify=True,
-                  rficlean=False,
-                  ndm=32,ntime_plot=64,
-                  nfreq_plot=32,save_data=True,
+def filplot_entry(trigger_dict, toslack=True, classify=True,
+                  rficlean=False, ndm=32, nfreq_plot=32, save_data=True,
                   fllisting=None):
+    """ Given datestring and trigger dictionary, run filterbank plotting, classifying, slack posting.
+    Returns figure filename and classification probability. 
+    
+    Parameters
+    ----------
+    datestr : str 
+        datestring of observation
```

Comment:
This is no longer a parameter of the function and should be removed from the docstring